### ISSUE TITLE
Export module not updating to current collection

### ISSUE THREAD

**Issue by u12206050** [directus/directus#12239]
### Preflight Checklist

- [X] I have completed all [Troubleshooting Steps](https://docs.directus.io/getting-started/support/#troubleshooting-steps).
- [X] I'm on [the latest version of Directus](https://github.com/directus/directus/releases).
- [X] There's [no other issue](https://github.com/directus/directus/issues?q=is%3Aissue) that already describes my problem.

### Describe the Bug

The new export function isn't updating when I navigate to another collection.

It seems to stay on the first collection I am on after a refresh.

![Uploading Screen Recording 2022-03-19 at 10.21.40.gif…]()

### To Reproduce

Navigate between collections and check what state of the export module

### What version of Directus are you using?

9.7.0

### What version of Node.js are you using?

16.4.0

### What database are you using?

mysql

### What browser are you using?

chrome

### What operating system are you using?

osx

### How are you deploying Directus?

GCP

**Comment by t7tran** [comment]
Confirmed the bug. In our case, a mysterious value populated in the sort field. It appears as `f` for one collection, and `n` for another.

![image](https://user-images.githubusercontent.com/76130324/159242993-b771b7b3-ffa7-48e2-a20a-95c8084c893f.png)

**Comment by azrikahar** [comment]
@t7tran thanks for confirming. The mysterious value does seem weird, would you mind testing the fix in #12274 to see does this mysterious value still show up? If it still does, then feel free to open a new issue for it as it might be a separate bug. Hopefully not though 🤞

**Comment by t7tran** [comment]
Yes, this is fixed with #12274. Not the random Sort Field value.
